### PR TITLE
UI improvement

### PR DESCRIPTION
**Pull request by zekth**
Improved UI + Fix SignOut method

### CODE REVIEW

**Comment by Ranthalion** [comment]
:+1:

File: assets/sass/style.scss [hunk 1]
```diff
@@ -7,6 +7,18 @@
...
+.pointer {
+  cursor: pointer;
+}
+
+.bottom-bar {
+  border-top: solid 1px #CCC;
+}
+
+.custom-select {
+  background: #FFF;
```
Comment (Ranthalion) [inline assets/sass/style.scss]: I found this difference between the bootstrap site styline and reef-pi styling for .custom-select. 
-webkit-appearance: none;

I don't think we should need to change with background: #FFF; I would prefer to find root cause of losing -webkit-appearance:none

Comment (Ranthalion) [reply assets/sass/style.scss]: Based on https://getbootstrap.com/docs/4.1/getting-started/webpack/ I think it needs postcss-loader

Comment (ranjib) [reply assets/sass/style.scss]: @Ranthalion merging this for the time being, if you find this is problematic or annoying, happy to revisit this

Comment (zekth) [reply assets/sass/style.scss]: I'll add post css loader so.

**Pull request merged**